index.html

```diff
@@ -5,8 +5,9 @@
 	<title>ownCloud Phoenix</title>
 	<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no">
 	<meta name="theme-color" content="#375f7E"/>
+	<link rel="stylesheet" href="dist/themes/owncloud.css" />
```

Comment:
so the theme is "hard-coded" in the index.html?

Reply:
No, we need a way to plug it there. If the theme still relies on the config.json, we can make use of that. Others suggested to chose the theme from the backend.

L#8 is a quick WIP solution ;-)